src/gui/update.rs

```diff
@@ -1833,14 +1833,18 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
             query_and_sort_catalog(ajour);
         }
         Message::Interaction(Interaction::CatalogSourceSelected(source)) => {
-            log::debug!("Interaction::CatalogResultSizeSelected({:?})", source);
+            log::debug!("Interaction::CatalogSourceSelected({:?})", source);
 
             // Catalog source
             if let CatalogSource::Choice(source) = source {
                 ajour.config.catalog_source = Some(source);
-                let _ = ajour.config.save();
+            } else {
+                ajour.config.catalog_source = None;
             }
```

Comment:
Just to confirm, if they select ALL we set the config to None because we unwrap none back as ALL? Makes sense!

Reply:
Correct, I'll however update this with a little more explanation because it might look a little strange.